metadata.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "hetzner-filebeats",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
You can't do a patch level version bump for a change breaks backward compatibility. If you want to do something like this you must support the old config and convert prospectors to input and check filebeats versions (since input only supported from 6.3)